CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ option(BUILD_AVDECC_INTERFACE_PCAP_DYNAMIC_LINKING "Pcap protocol interface uses
 option(BUILD_AVDECC_INTERFACE_MAC "Build the macOS native protocol interface (macOS only)." TRUE)
 option(BUILD_AVDECC_INTERFACE_PROXY "Build the proxy protocol interface." FALSE)
 option(BUILD_AVDECC_INTERFACE_VIRTUAL "Build the virtual protocol interface (for unit tests)." TRUE)
+if(NOT WIN32)
```

Comment:
Not sure if it's best to hide the options for Windows, or always expose the option and filter out windows a little bit down in the file (where other PI checks are done), with auto-disable on windows + message maybe.

I'm afraid of getting cmake errors if passing an option that do not exist on windows (vs auto-disable for unsupported platforms). Actually maybe it can be supported on windows later as well (at least the unix socket)